skiko/src/nativeMain/kotlin/org/jetbrains/skia/impl/Native.native.kt

```diff
@@ -284,3 +281,13 @@ private external fun initCallbacks(
     callVoid: COpaquePointer,
     dispose: COpaquePointer
 )
+
+/**
+ * C++ needs char* with zero byte at the end. So, we need to increase original string and write 0 at end.
```

Comment:
I would remove this comment. The function name is self-evident now

Reply:
I would keep it, but inside the function. Without it, it is not obvious why we add +1

Reply:
zero-terminated string is a C term, not everyone knows it

Reply:
> I would keep it, but inside the function

Actually, it is fine outside as well

Reply:
Added comment back

Reply:
Let's write outside comment as: 

/**
 * Converts String to zero-terminated utf-8 byte array.
*/

 and add the comment inside:

//  C++ needs char* with zero byte at the end. So we need to allocate array with an extra zero byte.